server/controllers/habit.controller.js

```diff
@@ -46,8 +46,16 @@ const toggleHabbitDate = async (req, res) => {
     res.send(habits)
 }
 
+const deleteHabit = async (req, res) => {
+    const habit = await Module.findOne({_id: req.params.habitID}).exec()
+    await Module.deleteOne({_id: req.params.habitID})
+    const habits = await Module.find({userID: habit.userID})
+    res.send(habits)
+}
+
```

Comment:
It might be better if you do something like this (taken from mine)

<img width="953" alt="Screen Shot 2023-08-06 at 6 10 51 PM" src="https://github.com/awctw/study-dash/assets/105681176/df62f20a-7ebe-4258-9140-ba641fd69571">